local_modules/bitcore-lib-xpi/index.d.ts

```diff
@@ -0,0 +1,390 @@
+declare module "bitcore-lib-xpi" {
```

Comment:
This module is maintained by another group. I put the type definitions elsewhere because of that, but I guess you need them here to move the cashweb stuff up?